api/design/types/type.go

```diff
@@ -244,6 +250,9 @@ var ResourceData = ResultType("application/vnd.hub.resource.data", "ResourceData
 	Attribute("hubURLPath", String, "Url path of the resource in hub", func() {
 		Example("hubURLPath", "tekton/task/buildah")
 	})
+	Attribute("hubRawURLPath", String, "Api url path to get the resource raw yaml", func() {
```

Comment:
we should remove `v1`

also, can you improve the message a bit - `path of the api to get the raw yaml of resource from hub apiserver`